grudge/array_context.py

```diff
@@ -258,6 +297,10 @@ def __call__(self, arg_id_to_arg) -> ArrayContainer:
                 allocator=self.actx.allocator,
                 input_args=input_args_for_prg)
 
+        t_end = time()
+
+        print(f"Fusion actx took: {t_end - t_start} secs")
+
```

Comment:
```suggestion
```
Sorry, this was a spurious diff that had sprung up into my tree at some point.

Reply:
Thanks, removed.